addons/website_livechat/static/src/web/persona_model_patch.js

```diff
@@ -0,0 +1,16 @@
+import { patch } from "@web/core/utils/patch";
+import { Persona } from "@mail/core/common/persona_model";
+import { deserializeDateTime } from "@web/core/l10n/dates";
+
+const { DateTime } = luxon;
+
+patch(Persona.prototype, {
+    get historyString() {
+        const history = [];
+        for (const h of this.history) {
+            const time = deserializeDateTime(h[1]).toLocaleString(DateTime.TIME_24_SIMPLE);
+            history.push(`${h[0]} (${time})`);
+        }
+        return history.join(" → ");
```

Comment:
And then in JS you should check if you still receive the old format and do nothing, or the new format and parse accordingly.